ontologies/semopenalex-ontology.ttl

```diff
@@ -0,0 +1,527 @@
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Work> .
+
+<https://semopenalex.org/class/Work> a owl:Class;
+  owl:equivalentClass fabio:Work;
+  rdfs:label "Work"@en .
+
+<https://semopenalex.org/property/hasAssociatedInstitution> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range <https://semopenalex.org/class/Institution> .
+
+<https://semopenalex.org/property/hasConcept> a owl:ObjectProperty;
+  rdfs:domain _:node61;
+  rdfs:range skos:Concept .
+
+<https://semopenalex.org/class/ConceptScore> a owl:Class;
+  rdfs:label "ConceptScore"@en .
+
+<https://semopenalex.org/property/hasConceptScore> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/ConceptScore> .
+
+cito:cites a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/Work> .
+
+<https://semopenalex.org/property/countsByYear> a owl:ObjectProperty;
+  rdfs:domain _:node64;
+  rdfs:range <https://semopenalex.org/class/CountsByYear> .
+
+<https://semopenalex.org/class/Concept> a owl:Class;
+  rdfs:subClassOf skos:Concept;
+  rdfs:label "Concept"@en .
+
+<https://semopenalex.org/class/Venue> a owl:Class;
+  rdfs:label "Venue"@en .
+
+<https://semopenalex.org/class/CountsByYear> a owl:Class;
+  rdfs:label "CountsByYear"@en .
+
+<https://semopenalex.org/property/hasHostVenue> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/HostVenue> .
+
+<https://semopenalex.org/class/HostVenue> a owl:Class;
+  rdfs:label "HostVenue"@en .
+
+<https://semopenalex.org/property/hasAlternativeHostVenue> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/HostVenue> .
+
+<https://semopenalex.org/property/hasVenue> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/HostVenue>;
+  rdfs:range <https://semopenalex.org/class/Venue> .
+
+<https://semopenalex.org/property/hasOpenAccess> a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range <https://semopenalex.org/class/OpenAccess> .
+
+<https://semopenalex.org/class/OpenAccess> a owl:Class;
+  rdfs:label "OpenAccess"@en .
+
+dbo:location a owl:ObjectProperty;
+  rdfs:domain <https://semopenalex.org/class/Institution>;
+  rdfs:range <https://semopenalex.org/class/Geo> .
+
+<https://semopenalex.org/class/Geo> a owl:Class;
+  rdfs:label "Geo"@en .
+
+fabio:hasPubMedId a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range xsd:string .
+
+fabio:hasPubMedCentralId a owl:DatatypeProperty;
+  rdfs:domain <https://semopenalex.org/class/Work>;
+  rdfs:range xsd:string .
+
+<https://semopenalex.org/property/worksCount> a owl:DatatypeProperty;
+  rdfs:domain _:node71;
```

Comment:
why are we using rdfs:domain and rdfs:range (and not SHACL shapes)? Is there a reason?